schema-guide.md

```diff
@@ -1205,9 +1205,9 @@ An entity can represent different types of entities, such as a team, an institut
     ```
     ```yaml
     authors:
-      - family-names: McClane
+      - family-names: Druskat
```

Comment:
I guess no 

Welcome to the party pal!!

for you then :)

hashtag nakatomi towers